Sources/Core/AccessTokenWallet.swift

```diff
@@ -27,6 +27,13 @@ class AccessTokenWallet {
   let graphConnectionProvider: GraphConnectionProviding
   let graphRequestPiggybackManager: GraphRequestPiggybackManaging.Type
 
+  /**
+   The global shared instance of AccessTokenWallet
+
+   Overriding this value may result in unexpected and unwanted behavior
+  */
+  static var shared: AccessTokenWallet = AccessTokenWallet()
```

Comment:
Do we imagine this needing to be changed by an external dev?

```suggestion
  private(set) static var shared: AccessTokenWallet = AccessTokenWallet()
```